module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -2060,10 +2071,57 @@ protected function getItemStatusesForBib($id, $checkHoldings)
             $statuses[] = $entry;
         }
 
+        // Add orders
+        foreach ($orders as $locationCode => $orderSet) {
+            $location = $this->translateLocation($orderSet[0]['location']);
+            $statuses[] = [
+                'id' => $id,
+                'item_id' => "ORDER_{$id}_$locationCode",
+                'location' => $location,
+                'callnumber' => $bibCallNumber,
+                'number' => '',
+                'status' => $this->mapStatusCode('Ordered'),
+                'reserve' => 'N',
+                'item_notes' => $this->getOrderMessages($orderSet),
+                'availability' => false,
+                'duedate' => '',
+                'barcode' => '',
+                'sort' => $sort--
+            ];
+        }
+
         usort($statuses, [$this, 'statusSortFunction']);
         return $statuses;
     }
 
+    /**
+     * Get textual messages for orders
+     *
+     * @param array $orders Orders
+     *
+     * @return array
+     */
+    protected function getOrderMessages(array $orders): array
+    {
+        $messages = [];
+        foreach ($orders as $order) {
+            $messages[] = $this->translate(
+                [
+                    'HoldingStatus',
+                    ($order['copies'] === 1 ? 'copy' : 'copies') . '_ordered_on_date'
```

Comment:
As discussed on a different recent PR, maybe it's better to use full translation strings here to ensure findability if we're searching to see where particular strings are used in the code.

Reply:
Right back at me :) Fixed!